Java/ds/MiddleOfLinkedList.java

```diff
@@ -0,0 +1,116 @@
+//Java program to find middle of LinkedList
+
+/*Possible Approaches: 
+i) Traverse List: Traverse list to find it's length, then scan list to locate n/2 node from the beginning.
+ii) Two - Pointer Approach: It is the efficient approach (only one scan). Use 2 pointers. Move one at twice the speed of second. 
+When first pointer reaches the end of list, second pointer will point the middle node of the LinkedList.
+
+Using the 2 pointer approach to find the middle element with Time Complexity: O(n) Space Complexity: O(1)					   		
+**/
```

Comment:
Remove this line as you already mentioned at the EOF of the code and Extra one *